components/tutorial/tutorial_tip.jsx

```diff
@@ -64,8 +61,16 @@ export default class TutorialTip extends React.Component {
         }
 
         this.closeRightSidebar();
-        this.toggle();
+        this.hide();
+
+        const step = PreferenceStore.getInt(Preferences.TUTORIAL_STEP, UserStore.getCurrentId(), 0);
```

Comment:
As noted in the summary, we now only progress on an explicit `next` (or opting out of the tutorial steps). Previously, hitting escape or clicking outside the tutorial box (accidentally, sometimes) would dismiss a (single-pane) tutorial step.